firewalld.te

```diff
@@ -69,6 +69,8 @@ kernel_read_network_state(firewalld_t)
 kernel_read_system_state(firewalld_t)
 kernel_rw_net_sysctls(firewalld_t)
 
+allow firewalld_t modules_object_t:dir { getattr open search read };
```

Comment:
Wouldn't "files_read_kernel_modules(firewalld_t)" be a better fit as it allows search for modules_object_t:dir?